.changeset/sharp-cups-juggle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@wso2is/core": patch
```

Comment:
Package name should be `identity-apps-core`.

Reply:
Addressed with 36505395f0de512fe8d583fe79b774db55c96e4b